tests/test_cli.py

```diff
@@ -1,9 +1,14 @@
 # Copyright Contributors to the Cellarium project.
 # SPDX-License-Identifier: BSD-3-Clause
 
+from __future__ import annotations
```

Comment:
I have though about using `from __future__ import annotations` and adding `if TYPE_CHECKING` clause ([PR for it](https://github.com/cellarium-ai/cellarium-ml/pull/248)). However, it seems that the future of `from __future__ import annotations` is uncertain and it might get deprecated. What this import does is it stringifies annotations, however, there was a backlash against it and a new method was proposed that will defer annotations (PEP 649). This new method is expected to be released next year.

Reply:
Yeah, I think it might be better to just wait for the new method and implement it once it is realeased.